nyaa/models.py

```diff
@@ -271,6 +276,23 @@ def by_category_ids(cls, main_cat_id, sub_cat_id):
         return cls.query.filter(cls.id == sub_cat_id, cls.main_category_id == main_cat_id).first()
 
 
+class Comment(db.Model):
+    __tablename__ = DB_TABLE_PREFIX + 'comments'
+
+    id = db.Column(db.Integer, primary_key=True)
```

Comment:
Maybe make the comment id a composite key of torrent_id + comment_id ?
So comments in a torrent start from 1